src/Dev/Constraint/SSListContains.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace SilverStripe\Dev\Constraint;
+
+use PHPUnit_Framework_ExpectationFailedException;
+use SilverStripe\Dev\SSListExporter;
+use SilverStripe\View\ViewableData;
+
+/**
+ * Constraint for checking if a SS_List contains items matching the given
+ * key-value pairs.
+ *
+ * Class SSListContainsOnly
```

Comment:
For consistency can we please remove `@package` and all the IDE-added headers to these classes? We are trying to arrange our PHP api docs (auto-built) by namespace rather than package these days.